src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -786,8 +786,10 @@ export const WorkflowView = _.flow(
         useCallCache,
         deleteIntermediateOutputFiles,
         useReferenceDisks,
-        retryWithMoreMemory,
-        retryMemoryFactor,
+        // Memory Retry will return
```

Comment:
I agree with all your bike shed design choices except can we make the doorknob a slightly different shape (seems more accurate to say it MAY return rather than WILL return).